### PR TITLE
FOR REVIEW ONLY - js-algorand-sdk v1.14.0

### PR DESCRIPTION
**Pull request by onetechnical**
# v1.14.0

## Added

- Add stateproof keyreg field (#463)

## Changed

- Implement C2C tests (#498)
- Moving from travis to circleci (#507)
- Add installation instructions for vite users (#512)
- Update langspec for TEAL 6 (#518)
- Docs for `lookupAssetByID` and `lookupAccountTransactions` (#516)
- Make FromObject parameter IntelliSense human readable (#528)
- Bump url-parse from 1.5.1 to 1.5.8 (#529)

## Fixed

- Use HTTP request format arg to determine response type (#532)
- Update chromedriver (#535)

### CODE REVIEW

**Review by jasonpaulos — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# v1.14.0-beta.1
```
Comment (jasonpaulos) [inline CHANGELOG.md:1]: minor comment: I thought we decided to keep beta version changes and add a new section for the final release. That's at least what we did for 1.13.0

Comment (onetechnical) [reply CHANGELOG.md:1]: Hmm, could go either way on this. I like clean release notes, but maybe we keep the changelog as is, but create final release notes for the github release that combines them?

Comment (onetechnical) [reply CHANGELOG.md:1]: Ah - never mind, I see what you mean. Keep the beta sections but add a final section that incorporates.

**Review by jasonpaulos — APPROVED** [review]
Thanks for addressing the changelog issue

**Review by algobarb — APPROVED** [review]
LGTM

**Review by egieseke — APPROVED** [review]

**Pull request merged**